WEBROOOOT/style.css

```diff
@@ -0,0 +1,93 @@
+nav {
```

Comment:
Love how you are indenting your rules throughout most of this file - makes it super easy to read through it.